### PR TITLE
General improvements to SRanipal page

### PR DESCRIPTION
**Pull request by kusomaigo**
- Bit easier to read / follow
- Add troubleshooting section similar to Quest Pro page
- Add better instructions for 1.3.1.1 installer workaround

### CODE REVIEW

**Review by regzo2 — APPROVED** [review]

File: docs/hardware/VIVE/sranipal.mdx [hunk 1]
```diff
@@ -1,60 +1,114 @@
...
 
-1. Install Vive Console (avaliable on Steam or Vive website)
+1. Use the installer of SRanipal v1.3.1.1
 2. Use the v1.3.6.5 zip of SRanipal in the VRCFT Discord #file-share channel
-3. Use the installer of SRanipal v1.3.1.1
+3. Install Vive Console (avaliable on Steam or Vive website)
 
 :::warning
-Version 1.3.2.0 is known to unnecessarily hog computer resources. We do not recommend using v1.3.2.0. 
+Version 1.3.2.0 is known to unnecessarily hog computer resources. We generally not recommend using v1.3.2.0.
```
Comment (benaclejames) [inline docs/hardware/VIVE/sranipal.mdx:45]: "We generally don't"

File: docs/hardware/VIVE/sranipal.mdx [hunk 2]
```diff
@@ -1,60 +1,114 @@
...
 :::
 
 :::warning
-Versions of SRanipal including and after **1.3.6.5** will generate *an unreasonable* amount of spam in it's EyeModule log when eye tracking is initialized. 
-Do not leave VRCFaceTracking unnecessarily running in the background for long periods of time as the size of the log file can easily get into the range of tens to hundreds **gigabytes**
-if left unchecked. 
-Until this is resolved, we recommend users use older versions of SRanipal if possible. 
+Versions of SRanipal including and after **1.3.6.5** will generate *an unreasonable* amount of spam in it's EyeModule log when eye tracking is initialized.
+Do not leave VRCFaceTracking unnecessarily running in the background for long periods of time as the size of the log file can easily get into the range of
+tens to hundreds **gigabytes** if left unchecked.
```
Comment (benaclejames) [inline docs/hardware/VIVE/sranipal.mdx:55]: "tens to hundreds of gigabytes"

File: docs/hardware/VIVE/sranipal.mdx [hunk 3]
```diff
@@ -1,60 +1,114 @@
...
+Until this is resolved, we recommend users use older versions of SRanipal if possible.
 :::
 
 The recommended version to use as of now (May 2023):
 
-- Most users (Vive Pro Eye, Vive Facial Tracker) should use the [v1.3.1.1 installer](#installing-via-v1311-installer) if possible
+- If you aren't using a Focus 3 (basically everyone), should use the [v1.3.1.1 installer](#installing-via-v1311-installer)
```
Comment (benaclejames) [inline docs/hardware/VIVE/sranipal.mdx:61]: "you should use the"

File: docs/hardware/VIVE/sranipal.mdx [hunk 4]
```diff
@@ -63,58 +117,54 @@ To use the SRanipal Module, you'll need a compatible headset and the [SRanipal R
...
-
-1. Download the [SRanipal v1.3.1.1 installer](https://drive.google.com/drive/folders/1hU1tvOiTHQbT-Ad-wBJ8II8uXkQ8bvvh?usp=share_link)
-2. Run the installer and complete the installation process
-3. Replace the SRanipal VRCFaceTracking module libs with the pre-Vive Focus 3 module versions.
+### Installing Via Vive Console
 
-**Important:** The installer for v1.3.1.1 has a known failure point during installation: **Error 1001**. It is not entirely clear why it happens.
+1. It is recommended to install [Vive Console via Steam](https://store.steampowered.com/app/1635730/VIVE_Console_for_SteamVR/)
+2. After install, run Vive Console once to let it's internal installers run. You never need to run Vive Console again.
+    - Vive Console does take up a lot of space (~1.6Gb), so copying the SRanipal folder (``Steam\steamapps\common\VIVEDriver\App\SRanipal``) out , then uninstalling Vive Console is another avenue. Note that this will require manual registration of the SRanipal service and drivers as the Vive Console uninstall will remove them.
```
Comment (benaclejames) [inline docs/hardware/VIVE/sranipal.mdx:124]: Consider using `:::note` admonition here plz

**Review by benaclejames — COMMENTED** [review]
Looks good! Just a few minitor formatting issues

**Pull request merged**